packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -48,10 +51,13 @@ export default function ProfileForm({
   ...props
 }: ProfileFormProps): ReactElement {
   const { user, updateUser } = useContext(AuthContext);
-
   const formRef = useRef<HTMLFormElement>(null);
 
-  const [userTimeZone, setUserTimeZone] = useState<string>(user?.timezone);
+  const emptyTimezoneMode =
+    mode === 'update' ? 'Europe/London' : getUserDefaultTimezone();
+  const initialTimezone = user?.timezone ? user?.timezone : emptyTimezoneMode;
```

Comment:
This gets calculated on every render although we need it just to set the state for the first time.
We can use the `useState` variation that gets a callback instead. Refactor this into a function outside the component and call it in the `useState`